### ISSUE TITLE
Guru Meditation Error when navigating in the cheat menu

### ISSUE THREAD

**Issue by ChampionLeake** [DS-Homebrew/TWiLightMenu#368]
When you press X to delete a game and cancel, things seem normal until you press Y to open game-settings and press X for the cheat menu. When you do anything while in the cheat menu, the user is prompted with the Guru Meditation Error.

This occurs for both DSi and 3DS consoles.

**Comment by bilyso** [comment]
I have been having the same issue. It seems to happen to me when I get to the bottom of the cheat list.

**Comment by bilyso** [comment]
DSI and 3DS for me

**Comment by Erimatai** [comment]
I'm having the same problem in my N2DS with the DSi theme in Twilight Menu++, from the N2DS SD card. My screenshots:
![kirby_crash 1](https://user-images.githubusercontent.com/12172565/52447964-5eae8200-2b00-11e9-8e42-2cc3463fb708.jpg)
![kirby_crash](https://user-images.githubusercontent.com/12172565/52447968-63733600-2b00-11e9-8d67-080b6931c615.jpg)

**Comment by NightYoshi370** [comment]
Just to make sure, where did you all download this?
I can't reproduce this at all

**Comment by Erimatai** [comment]
What file? The Twilight Menu++ or the usrcheat.dat file?

**Comment by bilyso** [comment]
I downloaded Twilight menu from github and I got the cheat file from https://gbatemp.net/threads/deadskullzjrs-flashcart-cheat-databases.488711/

**Comment by Erimatai** [comment]
I downloaded Twilight from github and "usrcheat.dat" from Filetrip. Somebody in some forum posted it and I downloaded many years ago. It's the same cheat file I used for years in my R4DS.

**Comment by ValliantStorme** [comment]
Same error, using latest release and DeadSkullzJr's cheat database. I opened the cheat list for Animal Crossing Wild World. After scrolling sufficiently far down the cheat list, attempting to exit the list caused a data abort.

**Comment by david4444** [comment]
Happened to me with the cheat database from the official updater by scrolling to the end of the list.

**Comment by NightYoshi370** [comment]
@david4444 @ValliantStorme @Erimatai @bilyso @ChampionLeake can you guys try the latest nightly of TWiLightMenu++ and see if that fixes your issue?

**Comment by david4444** [comment]
@NightYoshi370 I just tested with the nightly on the DSi theme, the crash is fixed. However you can't hold down up/down to scroll automatically anymore (Which i remember you could). You can press left or right which makes it easier.
I tested with the cheats database from the updater, on Kirby's Super Star Ultra, New Super Mario Bros and Newer Super Mario Bros. DS.

**Comment by NightYoshi370** [comment]
@david4444 hey, I just checked and it doesn't crash for me with Sonic Colors on the latest nightly.
Can you try again?

**Comment by david4444** [comment]
Sorry for the late response @NightYoshi370, I didn't get any notification.
Well, I tested it furthermore, was about to write about no crashes.

![really i was about to type how it didn't crash](https://user-images.githubusercontent.com/17904548/57194426-81a9d000-6f4f-11e9-81cc-56ec41781cba.jpeg)

**Comment by Erimatai** [comment]
Sorry for the long long reply, I was busy in my work. Well, I tested with the last version and seems no crash in the some games I showed. I didn't test games like @david4444 but I notice the cheats menu showed and closed so fast with a single X press in some games like Kirby Super Star Ultra and Contra 4 (and therefore I messed the cheats).

**Comment by qwertiyourip** [comment]
same issue, i'm on DSi with Unlaunch+HiyaCFW 
![IMG_20190515_001357](https://user-images.githubusercontent.com/26675407/57735927-7e78b780-76a6-11e9-8c25-c114d4566b2f.jpg)

**Comment by Clebana** [comment]
Hello everyone, I had this problem when I tried to add cheats to usrcheat.dat and when I tried to make my own .dat. Guru meditation appeared when pressing Y then X. Apparently, it was because when I used "save as" in r4cce (I use the french version, I guess you have "save as" in the english one), I choosed the wrong option ("Cheat database" instead of "R4 Cheat DB").
My problem might be different than yours. However, maybe it can help some people with the same issue.


(Sorry if my engligh isn't quite correct, I'm from Baguette-land)

![Capture](https://user-images.githubusercontent.com/38736767/57989382-38529800-7a9a-11e9-8168-ded98342fb74.PNG)
